src/connectionhealth.cpp

```diff
@@ -93,7 +93,6 @@ void ConnectionHealth::stop() {
   m_dnsPingTimer.stop();
 
   stopTimingDistributionMetric(m_stability);
-  setStability(Stable);
```

Comment:
Why do we need to change this file as part of this?

Reply:
Without this we won't detect the move from NoSignal back to Stable and know to hide the ServerUnavailable modal.